crates/modelardb_server/src/storage/mod.rs

```diff
@@ -78,10 +80,17 @@ pub(super) const QUERY_DATA_FOLDER_SCHEME_WITH_HOST: &str = "query://query";
 /// element is a [`Timestamp`] and a [`Value`](use crate::types::Value). Note that the resulting
 /// size of the buffer has to be a multiple of 64 bytes to avoid the actual capacity being larger
 /// than the requested due to internal alignment when allocating memory for the two array builders.
-const UNCOMPRESSED_DATA_BUFFER_CAPACITY: usize = 64 * 1024;
+pub static UNCOMPRESSED_DATA_BUFFER_CAPACITY: Lazy<usize> = Lazy::new(|| {
+    env::var("MODELARDBD_UNCOMPRESSED_DATA_BUFFER_CAPACITY").map_or(64 * 1024, |value| {
+        let parsed = value.parse::<usize>().unwrap();
 
-/// The number of bytes that are required before transferring a batch of data to the remote object store.
-const TRANSFER_BATCH_SIZE_IN_BYTES: usize = 64 * 1024 * 1024; // 64 MiB;
+        if parsed >= 64 && parsed % 64 == 0 {
+            parsed
+        } else {
+            panic!("MODELARDBD_UNCOMPRESSED_DATA_BUFFER_CAPACITY must be a multiple of 64.")
```

Comment:
should it panic or fall back to the closest multiple of 64?

Reply:
This panic would happen when they start the server so not much would be lost so I think it is fine to just panic.